.detekt.yml

```diff
@@ -13,3 +13,11 @@ style:
     active: false
   ProtectedMemberInFinalClass:
     active: false
+  UnusedPrivateMember:
+    active: false
+
+exceptions:
+  RethrowCaughtException:
+    active: false
+  SwallowedException:
+    active: false
```

Comment:
These were newly triggering some `detekt` failures. I've disabled the ones that are disabled in a-c, and fixed the others.